reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -649,13 +643,34 @@ private fun DIV.licenseLink(license: String) {
     }
 }
 
+private fun DIV.licensesLink(expression: SpdxExpression) {
+    when (expression) {
+        is SpdxLicenseIdExpression -> {
+            licenseLink(expression.toString())
+        }
+        is SpdxLicenseWithExceptionExpression -> {
+            licenseLink(expression.simpleLicense())
+            +" WITH "
+            licenseLink(expression.exception)
+        }
+        is SpdxCompoundExpression -> {
+            licensesLink(expression.left)
+            +" ${expression.operator} "
+            licensesLink(expression.right)
+        }
+        else -> {
+            +expression.toString()
+        }
+    }
+}
+
 private fun getLicenseResourcePath(license: String): String =
     when {
         license.startsWith(SpdxConstants.LICENSE_REF_PREFIX) -> {
             "/licenserefs/$license"
         }
         license.contains("exception") -> {
-            "/exception/$license"
+            "/exceptions/$license"
```

Comment:
This change belongs to the previous commit. And while at it, please make these cases one-liners by removing the curly braces.